src/modules/cvrp/processor/vrp_processor.py

```diff
@@ -1,20 +1,28 @@
 import time
+import requests
 from math import inf
+import json
 
 from ..vrp import VRP
 from ...utils.structures.parcel import Parcel
 from ...utils.structures.deliveries import Deliveries
 from ...utils.structures.plan import Plan
 from ...utils.structures.vehicle import Vehicle
 from ...create_graph.config.config_parser import ConfigParser
+from ...utils.structures.node import Node
+
+url = "https://graphhopper.com/api/1/vrp?key=e8a55308-9419-4814-81f1-6250efe25b5c"
```

Comment:
"key" should be in config.json and not in the code (repo). You can do it by adding new key to config.json called graphopper_key = "e8a55308-9419-4814-81f1-6250efe25b5c" and adding getter in config-parser.py to retrieve it.